src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/OffenderBookingBuilder.kt

```diff
@@ -98,19 +95,4 @@ class OffenderBookingBuilder(
   ) {
     this.incentives += IncentiveBuilder(iepLevelCode, userId, sequence, commentText, auditModuleName, iepDateTime)
   }
-
-  override fun courseAllocation(
-    courseActivity: CourseActivity,
-    startDate: String?,
-    programStatusCode: String,
-    endDate: String?,
-    payBands: MutableList<CourseAllocationPayBandBuilder>,
-    endReasonCode: String?,
-    endComment: String?,
-    attendances: MutableList<CourseAttendanceBuilder>,
-    dsl: CourseAllocationDsl.() -> Unit,
-  ) {
```

Comment:
Nothing is using the old style course allocation DSL so was able to remove entirely.